### DISCUSSION TITLE
Configure enviornment variable FASTF1_CACHE

### DISCUSSION THREAD

**Discussion by Casper-Guo** [theOehrly/Fast-F1#367] [Q&A] [answered]
Probably a trivial question but I am wondering how to configure this variable so FastF1 can use my existing cache directory even without calling `enable_cache`.

**Comment by theOehrly** [comment, marked as answer]
How you set an environment variable depends on your operating system. For example, on the Windows commandline, you can do ``set FASTF1_CACHE=C:\path\to\cache``. But that will not be persistent and only be valid for the currently open terminal.
But you can add permanent system-wide environment variables on Windows, Mac and Linux, so no matter where you use FastF1 on your PC, it will use the same cache directory.
